#! /bin/sh /usr/share/dpatch/dpatch-run
## 31_spelling.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix spelling errors in binary.

@DPATCH@
diff -urNad xfig~/f_load.c xfig/f_load.c
--- xfig~/f_load.c
+++ xfig/f_load.c
@@ -373,5 +373,5 @@
 	/* Format error; relevant error message is already delivered */
 	;
     else
-	file_msg("File \"%s\" is not accessable; %s.", file, strerror(err));
+	file_msg("File \"%s\" is not accessible; %s.", file, strerror(err));
 }
diff -urNad xfig~/f_readgif.c xfig/f_readgif.c
--- xfig~/f_readgif.c
+++ xfig/f_readgif.c
@@ -137,7 +137,7 @@
 
 		if (c == '!') { 		/* Extension */
 			if (! ReadOK(file,&c,1))
-				file_msg("GIF read error on extention function code");
+				file_msg("GIF read error on extension function code");
 			(void) DoGIFextension(file, c);
 			continue;
 		}
